test/e2e-encryption-kms/encryption_kms.go

```
@@ -0,0 +1,123 @@
+	// Deploy the mock KMS plugin for testing.
+	// NOTE: This manual deployment is only required for KMS v1. In the future,
+	// the platform will manage the KMS plugins, and this code will no longer be needed.
+	librarykms.DeployUpstreamMockKMSPlugin(context.Background(), t, library.GetClients(t).Kube, librarykms.WellKnownUpstreamMockKMSPluginNamespace, librarykms.WellKnownUpstreamMockKMSPluginImage, librarykms.DefaultKMSPluginCount)
+
+	ctx := context.TODO()
+	cs := operatorencryption.GetClients(t)
+
+	ns := fmt.Sprintf("test-kms-encryption-on-off-%d", rand.IntN(4))
+	_, err := cs.KubeClient.CoreV1().Namespaces().Create(ctx, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: ns}}, metav1.CreateOptions{})
+	require.NoError(t, err)
+	defer cs.KubeClient.CoreV1().Namespaces().Delete(ctx, ns, metav1.DeleteOptions{})
```

golangci-lint (2.12.2)

[error] Error return value of `(k8s.io/client-go/kubernetes/typed/core/v1.NamespaceInterface).Delete` is not checked

(errcheck)